build/Jenkinsfile

```diff
@@ -20,19 +20,38 @@ pipeline {
     stages {
         stage('Install') {
             steps {
-                sh 'make node_modules'
+                ansiColor('xterm') {
+                    sh 'make node_modules'
+                }
             }
         }
 
         stage('Check Style') {
             steps {
-                sh 'make check-style'
+                ansiColor('xterm') {
+                    sh 'make check-style'
+                }
+            }
+        }
+
+        stage('Check i18n') {
+            steps {
+                ansiColor('xterm') {
+                    sh """
+                        cp i18n/en.json /tmp/en.json
+                        git clone https://github.com/mattermost/mattermost-mobile /tmp/mattermost-mobile
```

Comment:
we should not check the strings of the mobile repo here

Reply:
i did not add that. maybe @jespino will revert that

Reply:
Yes I did, normally this command receive both the mobile repo and the webapp repo, but it's true that is not needed for the extract. I going to remove that.

Reply:
Done. I don't want to change the mattermost-utilities because makes sense to reuse the existing string on mobile when you extract the translations.